pkg/auth/serviceaccount_test.go

```diff
@@ -60,7 +60,7 @@ func TestGetOIDCServiceAccountNameForResource(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			if got := GetOIDCServiceAccountNameForResource(tt.gvk, tt.objectMeta); got != tt.want {
-				t.Errorf("GetServiceAccountName() = %v, want %v", got, tt.want)
+				t.Errorf("GetOIDCServiceAccountNameForResource() = %v, want %v", got, tt.want)
```

Comment:
@MeenuyD I think you need to change the `want` fields in the array of tests above. If you look at the test failures [here](https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_eventing/7471/unit-tests_eventing_main/1728054320873607168), you'll see that they are failing because you changed the format of the string being returned by the function, and they are expecting the old format. So, just update the expected values and they should pass!